src/System.Net.Sockets/ref/System.Net.Sockets.cs

```diff
@@ -483,11 +446,7 @@ public enum TransmitFileOptions
     public partial class UdpClient : System.IDisposable
     {
         public UdpClient() { }
-        public UdpClient(int port) { }
-        public UdpClient(int port, System.Net.Sockets.AddressFamily family) { }
-        public UdpClient(System.Net.IPEndPoint localEP) { }
         public UdpClient(System.Net.Sockets.AddressFamily family) { }
-        public UdpClient(string hostname, int port) { }
```

Comment:
@josguil We need at least one of the ctors to Bind the UDP connectionless socket. Otherwise ReceiveAsync won't be able to function at all.
The underlying Win32 API shouldn't be blocking since no actual I/O is performed.